harvester/core/migrations/0027_query_annotations.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 3.2.8 on 2021-11-24 11:16
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('core', '0026_alter_objectiveproperty_property'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Query',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('query', models.CharField(db_index=True, max_length=255)),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('modified_at', models.DateTimeField(auto_now=True)),
+            ],
+            options={
+                'verbose_name_plural': 'queries',
+            },
+        ),
+        migrations.CreateModel(
+            name='QueryRanking',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('subquery', models.CharField(db_index=True, max_length=255)),
+                ('ranking', models.JSONField(default=dict)),
```

Comment:
waarom is dit json?

Reply:
Dan kunnen we het zonder modificatie doorsturen naar eval van ES

Reply:
Ja ok, omdat we uiteindelijk toch nog deels die json opbouwen zag ik het nut niet, maar ook niet perse erg 😉